index.src.html

```diff
@@ -225,14 +225,8 @@ <h2>
       </p>
       <p>
         The <dfn data-cite=
-        "!GETUSERMEDIA#idl-def-ConstrainDOMString"><code>ConstrainDOMString</code></dfn>,
```

Comment:
Can you review all the dependencies and remove all that are no longer used?

Reply:
Done in [Consolidate the dependency list.](https://github.com/w3c/mediacapture-depth/pull/168/commits/af4fe13525a00e0655d8a0d1c845228e55245b90) patch.